Base/usr/share/man/man1/lsof.md

```diff
@@ -1,26 +1,42 @@
 ## Name
 
-lsof
+lsof - List open files
 
 ## Synopsis
 
-```sh
-$ lsof [-p pid] [-d fd] [-u login/UID] [-g PGID] [filename]
+```**sh
+$ lsof [options]... [filename]
 ```
 
 ## Description
 
-List open files of a processes. This can mean actual files in the file system, sockets, pipes, etc.
+`lsof` lists opened files by certain processes, such as pipes, sockets, regular files...
+If `filename` is specified, `lsof` will filter out by processes having `filename` opened.
+
+## Options
+
+* `-p pid`: Filter by PID
+* `-d fd`: Filter by file descriptor
+* `-u login/UID`: Filter by login/UID
+* `-g PGID`: Filter by process group ID
```

Comment:
I feel like if PGID is capitalized here, then PID and FD should be capitalized above

Reply:
I left them lowercase in utmpupdate. Should I make them uppercase there too? I personally prefer lowercase.

Reply:
Also `username` sounds better than `login`,  I changed that.